pinot-core/src/main/java/com/linkedin/pinot/core/indexsegment/IndexSegment.java

```diff
@@ -17,7 +17,7 @@
 
 import com.linkedin.pinot.common.segment.SegmentMetadata;
 import com.linkedin.pinot.core.common.DataSource;
-import com.linkedin.pinot.core.startree.StarTree;
+import com.linkedin.pinot.core.startree.StarTreeInterf;
```

Comment:
Yes, I prefer the same. However, doing so breaks backward compatibility, ie pre-existing indexes  cannot be de-serialized once class name is changed.